lib/licenceListSort.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (licenceArray) => {
+    let listOfLicences = [];
+
+    if (!Array.isArray(licenceArray)) {
+        return licenceArray;
+    }
+
+    licenceArray.forEach(function(item) {
```

Comment:
No big deal, but maybe this can be done using `map`
```
const listOfLicences = licenceArray.map((item) => {
    return {
        licenceId:item.id,
        creationDate: item.creationDateTime
    };
});
```